index.html

```diff
@@ -1,21 +1,43 @@
 <!DOCTYPE html>
 <html data-ng-app="wdiRadio">
-  <head>
-    <title>WDI Radio</title>
-    <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.3.15/angular.min.js"></script>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/angular-ui-router/0.2.15/angular-ui-router.min.js"></script>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/angular.js/1.5.0-beta.2/angular-resource.min.js"></script>
+<head>
+  <title>WDI Radio</title>
+  <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.3.15/angular.min.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/angular-ui-router/0.2.15/angular-ui-router.min.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/angular.js/1.5.0-beta.2/angular-resource.min.js"></script>
 
-    <script src="js/app.js"></script>
-    <script src="js/songs/songs.module.js"></script>
-    <script src="js/songs/index.controller.js"></script>
-    <script src="js/songs/song.factory.js"></script>
-    <script src="js/songs/form.directive.js"></script>
-    <script src="js/songs/player.directive.js"></script>
+  <script src="bower_components/angular-cookie/angular-cookie.js"></script>
+  <script src="bower_components/ng-token-auth/dist/ng-token-auth.js"></script>
 
-  </head>
-  <body>
-    <h1>WDI Radio</h1>
-    <main data-ui-view></main>
-  </body>
+  <script src="js/app.js"></script>
+  <script src="js/songs/songs.module.js"></script>
+  <script src="js/songs/index.controller.js"></script>
+  <script src="js/songs/song.factory.js"></script>
+  <script src="js/songs/form.directive.js"></script>
+  <script src="js/songs/player.directive.js"></script>
+
+</head>
+<body>
+  <h1>WDI Radio</h1>
+
+  <div ng-controller="LoginCtrl as vm">
+    <button ng-click="vm.signOut()">Sign Out</button>
```

Comment:
This shows by default when running the app the first time. Could be confusing for user.